core/services/fluxmonitorv2/integrations_test.go

```diff
@@ -414,15 +415,17 @@ func assertPipelineRunCreated(t *testing.T, db *sqlx.DB, roundID int64, result f
 	return run
 }
 
-func checkLogWasConsumed(t *testing.T, fa fluxAggregatorUniverse, db *sqlx.DB, pipelineSpecID int32, blockNumber uint64) {
+func checkLogWasConsumed(t *testing.T, fa fluxAggregatorUniverse, db *sqlx.DB, pipelineSpecID int32, blockNumber uint64, cfg pg.LogConfig) {
```

Comment:
this likely applies to many other places as well, but couldn't we create some struct for both  `db *sqlx.DB` and `cfg pg.LogConfig` ? to avoid passing the config separately in many cases

Reply:
Actually, we discussed this already and Jordan was suggesting to not do that until we have a DI integrated, which should address this issue as well. 
https://app.shortcut.com/chainlinklabs/story/11829/use-dependency-injection-in-application-startup
What do you think?